src/StreamFetcher.ts

```diff
@@ -11,111 +11,126 @@ const logger = getLogger('streamr:StreamFetcher')
 const MAX_AGE = 15 * 60 * 1000 // 15 minutes
 const MAX_AGE_MINUTE = 1000 // 1 minutes
 
-function formHeaders(sessionToken: string|undefined) {
-    const headers: any = {}
+function formHeaders(sessionToken?: string) {
+    const headers: Headers = new Headers()
     if (sessionToken) {
-        headers.Authorization = `Bearer ${sessionToken}`
+        headers.set('Authorization', `Bearer ${sessionToken}`)
```

Comment:
TypeScript will complain "key Authorization does not exist in type InitHeaders" when I tried to type the `{}`. This seems to work.